installers/macOS/setup.py

```diff
@@ -103,7 +103,7 @@ def disk_image_name(make_lite=False):
     """
     Return disk image name
     """
-    dmg_name = f'Spyder-{SPYVER}_{machine()}'
+    dmg_name = f'Spyder-{machine()}'
```

Comment:
I think we agreed to not have `machine` here if `machine == 'x86_64'`, right?

Reply:
Yes on 5.x, for backward compat of the update check—this was changed in the update check itself, but evidently not here where the DMG is generated, so presumably similar logic should be adopted?

Reply:
Also, would need to be `_` instead of `-` to be match with our existing release naming for macOS in 5.5.1 (even if this _is_ internally inconsistent for now), otherwise we'll need to rename and reupload our existing 5.5.1 ARM64 installers too and tweak that bit in the current website and docs links (on top of removing the arch from the Intel installer name).

Reply:
Yep. Missed it, thanks.